chat-commands.js

```diff
@@ -1550,7 +1559,7 @@ exports.commands = {
 		Punishments.lock(targetUser, null, null, target);
 		return true;
 	},
-	lockhelp: ["/lock OR /l [username], [reason] - Locks the user from talking in all chats. Requires: % @ * & ~"],
+	lockhelp: ["/lock OR /l [username], [reason] spoiler: [proof/evidence] - Locks the user from talking in all chats. Requires: % @ * & ~"],
```

Comment:
`[...]` does not mean "optional" in PS help text.

What we'd usually do is just have two lines, one with, one without.